packages/server-wallet/e2e-test/e2e.test.ts

```diff
@@ -65,31 +78,60 @@ describe('e2e', () => {
     expect(channel.status).toBe('opening');
     expect(channel.turnNum).toBe(0);
   });
+});
 
-  it('can update pre-existing channel, send signed state via http', async () => {
-    const seed = withSupportedState(
-      {turnNum: 3},
-      {
-        channelNonce: 123456789, // something unique for this test
-        participants: [ping, pong],
-      }
-    )();
-
-    await Channel.query().insert([seed]); // Fixture uses alice() default
-    await Channel.bindKnex(knexPong)
-      .query()
-      .insert([{...seed, signingAddress: pong.signingAddress}]);
+it('can update pre-existing channel, send signed state via http', async () => {
+  // SETUP
+  const [ping, pong] = [aliceP(), bobP()];
+  const seed = withSupportedState(
+    {turnNum: 3},
+    {
+      channelNonce: 123456789, // something unique for this test
+      participants: [ping, pong],
+    }
+  )();
+
+  await ChannelPing.query().insert([seed]); // Fixture uses alice() default
+  await ChannelPong.query().insert([{...seed, signingAddress: pong.signingAddress}]);
+
+  const {channelId} = seed;
+
+  const expectSupportedState = async (C: typeof Channel, turnNum: number): Promise<any> =>
+    expect(C.forId(channelId, undefined).then(c => c.protocolState)).resolves.toMatchObject({
+      latest: {turnNum},
+    });
 
-    const {channelId} = seed;
+  await expectSupportedState(ChannelPing, 3);
+  await expectSupportedState(ChannelPong, 3);
+  // END SETUP
+
+  // SCRIPT
+  const pingScript = childProcess.spawn(`yarn`, [
```

Comment:
This could probably be moved to a function in `e2e-utils`

Reply:
Done: https://github.com/statechannels/statechannels/pull/2417/commits/69f358db00b84196bd84518ac143a2fffd6c4065